cobalt/dom/html_media_element.h

```diff
@@ -313,7 +316,7 @@ class HTMLMediaElement : public HTMLElement,
   // Time has not changed since sending an "ended" event.
   bool sent_end_event_;
 
-  scoped_refptr<MediaError> error_;
+  scoped_refptr<MediaError> error_;  // See SetError().
```

Comment:
Should we move the comment above the code?

Reply:
Done.